.github/dependabot.yaml

```diff
@@ -110,5 +129,7 @@ updates:
     commit-message:
       prefix: "chore"
     labels: []
-    ignore:
-      - dependency-name: "coder/coder"
```

Comment:
Why was this removed?

Reply:
Because IMO for dogfood we should always have the latest terraform provider. For example templates, we are ignoring and prefer manual updates.

Reply:
Normally changes to our Terraform provider come with added features, changes, or deprecations that need to be added to the example templates as well. I think it makes sense to do these changes at the same time.

Reply:
Sure. I agree. I will revert this.